### PR TITLE
docs: clean-up old references

### PR DESCRIPTION
**Pull request by jrasm91**
Remove some old reverse geocoding references from the docs.

### CODE REVIEW

File: docs/docs/install/docker-compose.mdx [hunk 1]
```diff
@@ -2,6 +2,9 @@
 sidebar_position: 30
 ---
 
+import CodeBlock from '@theme/CodeBlock';
+import ExampleEnv from '!!raw-loader!../../../docker/example.env';
```
Comment (jrasm91) [inline docs/docs/install/docker-compose.mdx:6]: This just inlines the actual file contents directly in the docs.

Comment (bo0tzz) [reply docs/docs/install/docker-compose.mdx:6]: That's freaking awesome

**Review by bo0tzz — APPROVED** [review]

File: docs/docs/install/docker-compose.mdx (hunk 1 above)
Comment (danieldietzler) [reply docs/docs/install/docker-compose.mdx:6]: That's freaking wild syntax!

**Review by danieldietzler — APPROVED** [review]
LGTM!

**Pull request merged**